### PR TITLE
Patternfly react core bump - remove obsolete overrides

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]
Assuming green, this looks nice. Thanks!

File: pkg/networkmanager/firewall.jsx [hunk 1]
```diff
@@ -836,7 +836,7 @@ class ActivateZoneModal extends React.Component {
                                 { customZones.map(z =>
                                     <Radio key={z} id={z} name="zone" value={z}
                                            isChecked={this.state.zone == z}
-                                           onChange={(value, e) => this.onChange("zone", e.target.value)}
+                                           onChange={e => this.onChange("zone", e.target.value)}
```
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:839]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/firewall.jsx [hunk 2]
```diff
@@ -878,13 +878,13 @@
                                isChecked={this.state.ipRange == "ip-entire-subnet"}
                                value="ip-entire-subnet"
                                id="ip-entire-subnet"
-                               onChange={(value, e) => this.onChange("ipRange", e.target.value)}
+                               onChange={e => this.onChange("ipRange", e.target.value)}
```
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:881]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx [hunk 3]
```diff
@@ -270,19 +270,19 @@ const AutoUpdatesDialog = ({ backend }) => {
             <Form isHorizontal onSubmit={save}>
                 <FormGroup fieldId="type" label={_("Type")} hasNoPaddingTop>
                     <Radio isChecked={!enabled}
-                           onChange={e => { setEnabled(false); setType(null) }}
+                           onChange={() => { setEnabled(false); setType(null) }}
```
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:273]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx [hunk 4]
```diff
@@ -270,19 +270,19 @@
...
                            isDisabled={pending}
                            label={_("No updates")}
                            id="no-updates"
                            name="type" />
                     <Radio isChecked={enabled && type === "security"}
-                           onChange={e => { setEnabled(true); setType("security") }}
+                           onChange={() => { setEnabled(true); setType("security") }}
```
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:279]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx [hunk 5]
```diff
@@ -270,19 +270,19 @@
...
                            isDisabled={pending}
                            label={_("Security updates only")}
                            id="security-updates"
                            name="type" />
                     <Radio isChecked={enabled && type === "all"}
-                           onChange={e => { setEnabled(true); setType("all") }}
+                           onChange={() => { setEnabled(true); setType("all") }}
```
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:285]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by garrett — APPROVED** [review]
:+1: Thanks! Looks good!

The firewall page headings are an improvement too! (I initially mixed up before/after.)

**Review by martinpitt — APPROVED** [review]
This is a nice improvement, thanks!

File: pkg/networkmanager/bond.jsx [hunk 6]
```diff
@@ -168,13 +168,13 @@
...
                 <FormGroup fieldId={idPrefix + "-mode-select"} label={_("Mode")}>
-                    <FormSelect id={idPrefix + "-mode-select"} onChange={setMode}
+                    <FormSelect id={idPrefix + "-mode-select"} onChange={(_, val) => setMode(val)}
                                 value={mode}>
                         {bond_mode_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>
                 {mode == "active-backup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
-                    <FormSelect id={idPrefix + "-primary-select"} onChange={setPrimary}
+                    <FormSelect id={idPrefix + "-primary-select"} onChange={(_, val) => setPrimary(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/bond.jsx:177]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/firewall.jsx [hunk 7]
```diff
@@ -836,7 +834,7 @@ class ActivateZoneModal extends React.Component {
                                 { customZones.map(z =>
                                     <Radio key={z} id={z} name="zone" value={z}
                                            isChecked={this.state.zone == z}
-                                           onChange={(value, e) => this.onChange("zone", e.target.value)}
+                                           onChange={e => this.onChange("zone", e.target.value)}
```
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:837]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/firewall.jsx [hunk 8]
```diff
@@ -878,13 +876,13 @@
                                isChecked={this.state.ipRange == "ip-entire-subnet"}
                                value="ip-entire-subnet"
                                id="ip-entire-subnet"
-                               onChange={(value, e) => this.onChange("ipRange", e.target.value)}
+                               onChange={e => this.onChange("ipRange", e.target.value)}
```
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:879]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx [hunk 9]
```diff
@@ -159,19 +159,19 @@ export const TeamDialog = ({ connection, dev, settings }) => {
                     <MemberInterfaceChoices idPrefix={idPrefix} memberChoices={memberChoices} setMemberChoices={setMemberChoices} model={model} group={connection} />
                 </FormGroup>
                 <FormGroup fieldId={idPrefix + "-runner-select"} label={_("Runner")}>
-                    <FormSelect id={idPrefix + "-runner-select"} onChange={setRunner}
+                    <FormSelect id={idPrefix + "-runner-select"} onChange={(_, val) => setRunner(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:162]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx [hunk 10]
```diff
@@ -159,19 +159,19 @@
...
                                 value={runner}>
                         {team_runner_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>
                 {(runner == "loadbalance" || runner == "lacp") && <FormGroup fieldId={idPrefix + "-balancer-select"} label={_("Balancer")}>
-                    <FormSelect id={idPrefix + "-balancer-select"} onChange={setBalancer}
+                    <FormSelect id={idPrefix + "-balancer-select"} onChange={(_, val) => setBalancer(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:168]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx [hunk 11]
```diff
@@ -159,19 +159,19 @@
...
                                 value={balancer}>
                         {team_balancer_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>}
                 {runner == "active-backup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
-                    <FormSelect id={idPrefix + "-primary-select"} onChange={setPrimary}
+                    <FormSelect id={idPrefix + "-primary-select"} onChange={(_, val) => setPrimary(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx [hunk 12]
```diff
@@ -182,7 +182,7 @@
                     </FormSelect>
                 </FormGroup>}
                 <FormGroup fieldId={idPrefix + "-link-watch-select"} label={_("Link watch")}>
-                    <FormSelect id={idPrefix + "-link-watch-select"} onChange={setLinkWatch}
+                    <FormSelect id={idPrefix + "-link-watch-select"} onChange={(_, val) => setLinkWatch(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:185]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx [hunk 13]
```diff
@@ -200,7 +200,7 @@
                         <TextInput id={idPrefix + "-ping-interval-input"} className="network-number-field" value={pingInterval} onChange={setPingInterval} />
                     </FormGroup>
                     <FormGroup fieldId={idPrefix + "-ping-target-input"} label={_("Ping target")}>
-                        <TextInput id={idPrefix + "-ping-target-input"} value={pingTarget} onChange={setPingTarget} />
+                        <TextInput id={idPrefix + "-ping-target-input"} value={pingTarget} onChange={(_, val) => setPingTarget(val)} />
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:203]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:273]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 4 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:279]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 5 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:285]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx [hunk 14]
```diff
@@ -296,7 +296,7 @@
                             <FormSelect id="auto-update-day"
                                          isDisabled={pending}
                                          value={day == "" ? "everyday" : day}
-                                         onChange={d => setDay(d == "everyday" ? "" : d) }>
+                                         onChange={(_, d) => setDay(d == "everyday" ? "" : d) }>
```
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:299]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/timer-dialog.jsx [hunk 15]
```diff
@@ -205,7 +205,7 @@ const CreateTimerDialogBody = ({ owner }) => {
                                        onChange={setDelayNumber} />
                             <FormSelect className="delay-unit"
                                         value={delayUnit}
-                                        onChange={setDelayUnit}
+                                        onChange={(_, val) => setDelayUnit(val)}
```
Comment (cockpituous) [inline pkg/systemd/timer-dialog.jsx:208]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/account-create-dialog.js [hunk 16]
```diff
@@ -112,7 +112,7 @@ function AccountCreateBody({ state, errors, change, shells }) {
             <FormGroup label={_("Authentication")} fieldId="accounts-create-locked" hasNoPaddingTop>
                 <Radio id="account-use-password"
                        label={_("Use password")}
-                       isChecked={!locked} onChange={checked => change("locked", !checked)}
+                       isChecked={!locked} onChange={(_, checked) => change("locked", !checked)}
```
Comment (cockpituous) [inline pkg/users/account-create-dialog.js:115]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/bond.jsx [hunk 17]
```diff
@@ -168,13 +168,13 @@ export const BondDialog = ({ connection, dev, settings }) => {
                     <MacMenu idPrefix={idPrefix} model={model} mac={mac} setMAC={setMAC} />
                 </FormGroup>
                 <FormGroup fieldId={idPrefix + "-mode-select"} label={_("Mode")}>
-                    <FormSelect id={idPrefix + "-mode-select"} onChange={setMode}
+                    <FormSelect id={idPrefix + "-mode-select"} onChange={(_, val) => setMode(val)}
```
Comment (cockpituous) [inline pkg/networkmanager/bond.jsx:171]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/bond.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/networkmanager/bond.jsx:177]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/firewall.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:837]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/firewall.jsx (hunk 8 above)
Comment (cockpituous) [inline pkg/networkmanager/firewall.jsx:879]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx (hunk 9 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:162]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx (hunk 10 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:168]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx (hunk 11 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx (hunk 12 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:185]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/team.jsx (hunk 13 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:203]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:273]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 4 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:279]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 5 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:285]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/packagekit/autoupdates.jsx (hunk 14 above)
Comment (cockpituous) [inline pkg/packagekit/autoupdates.jsx:299]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/timer-dialog.jsx (hunk 15 above)
Comment (cockpituous) [inline pkg/systemd/timer-dialog.jsx:208]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/account-create-dialog.js (hunk 16 above)
Comment (cockpituous) [inline pkg/users/account-create-dialog.js:115]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**